bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -389,4 +389,10 @@
 		<description>Operating mode of the Somfy thermostatic valve</description>
 		<state readOnly="true"/>
 	</channel-type>
+
+	<channel-type id="scenes">
+		<item-type>String</item-type>
+		<label>Scenes</label>
+		<description>The scenes defined in the cloud portal</description>
+	</channel-type>
```

Comment:
Tahoma users knows ```scenarios``` and Tahoma internally uses ```actionsgroups```. Please do not introduce ```scenes``` and please rename the channel to ```scenarios```.

Reply:
I just used scene as I have the feeling that scene is the correct english word for what is a "scenario" in French.

I just found that:: "As nouns the difference between scene and scenario is that scene is the location of an event that attracts attention while scenario is an outline of the plot of a dramatic or literary work. "
So yes, scenario is the right word to use I think (now).

Reply:
good to know it :-) So the Tahoma guys have translated it right

Reply:
Done